doc/sphinx/matlab/onedim.rst

```diff
@@ -1,5 +1,5 @@
 .. default-domain:: mat
-.. currentmodule:: OneDim
+.. currentmodule:: OneDim.+ct
```

Comment:
```suggestion
.. currentmodule:: +ct.+oneD
```
Going with namespaces, I want to suggest to introduce a sub namespace `oneD` (i.e. following C++ folder nomenclature).

Members would then be accessed as
```
>>> ct.oneD.CounterFlowDiffusionFlame(left, flow, right)
```